src/styles/_typography.scss

```diff
@@ -1,4 +1,5 @@
 // Font Size
+$size-xlarge: 2rem; // 32px
```

Comment:
Thinking of deprecating these font sizes, as they aren't defined in the design system.

Reply:
But there is 32px font size in buy domain design

Reply:
I don't think Figma variables have been set up for font-sizes, so we don't get any benefit from storing them as SCSS variables.